src/main/java/com/endava/soa_5ed/task4/collections/UsersFilter.java

```diff
@@ -0,0 +1,30 @@
+package com.endava.soa_5ed.task4.collections;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class UsersFilter {
```

Comment:
If this is a kind o Utility Class, you may create a private constructor and make the methods static.

Reply:
Done